components/frontend/guestRoomDetailsPage/SimilarRooms.tsx

```diff
@@ -5,41 +5,20 @@ import { roomsData } from '@/data/roomDetailsPage';
 import { similarRoomsDetails } from '@/data/roomDetailsPage';
 import { lora, mulish } from "../../../app/fonts";
 
-const titleStyle = {
-  color: "#11142D",
-  fontSize: "39px",
-  fontStyle: "normal",
-  fontWeight: 700,
-  lineHeight: "120%",
-  letterSpacing: "0.195px",
-};
-
-const descriptionStyle = {
-  color: "var(#515151",
-  fontSize: "16px",
-  lineHeight: "125%",
-  fontStyle: "normal",
-  letterSpacing: "0.08px",
-  fontWeight: "400",
-  marginTop: "15px",
-  marginBottom: "35px",
-  width: { xs: "none", sm: "504px", md: "504px" },
-};
-
 const SimilarRooms = () => {
   return (
     <>
-      <Typography variant="h3" className={lora.className} sx={titleStyle}>
+      <Typography   color="#11142D" fontSize= "39px" fontStyle= "normal" fontWeight= "700" lineHeight= "120%" letterSpacing= "0.195px">
```

Comment:
Avoid using %. You can use lineHeight: 1.2 instead. Also, why have you not added these values to theme?